src/openforms/js/components/admin/forms/objects_api/ObjectTypeSelect.js

```diff
@@ -94,6 +95,7 @@ const ObjectTypeSelect = ({
             const okToProceed = onChangeCheck === undefined || onChangeCheck();
             if (okToProceed) setValue(selectedOption.value);
           }}
+          {...selectProps}
```

Comment:
Yesterday this issue popped back into my mind, and took a look. Maybe [Portaling](https://react-select.com/advanced#portaling) is a good solution. It allows us to move the dropdown to a different part of the dom tree, solving parent-child styling issues (and would probably save us some z-index magic styling).

(As i'm writing this, I do wonder if this would be a big accessibility issue... 🤔. I haven't looked yet how this works for screen readers and tab navigation... (I think the menu receives focus when opened, so maybe it won't be an issue...?))

Reply:
I can't imagine it *not* messing with the tab order :/

Reply:
Keyboard navigation is fine, and Robin verified the screenreader support which also appears to be fine.

Portaling appears to be the only viable solution here, so it's been adapted, let's see if it breaks anything else :(